Sources/Components/SortView.swift

```diff
@@ -0,0 +1,78 @@
+import SwiftUI
+
+struct SortView: View {
+    @State
+    var starSort = SortBy.starsAsc
+    @State
+    var forkSort = SortBy.forksAsc
+    @State
+    var updateSort = SortBy.updatedAsc
+
+    var actionHandler: ((SortBy) -> Void)?
+
+    var body: some View {
+        HStack {
+            Text("Sort by:")
+                .font(.headline)
+            Spacer()
+
+            Button {
+                if starSort == .starsAsc {
+                    starSort = .starsDes
+                } else {
+                    starSort = .starsAsc
+                }
+                actionHandler?(starSort)
+            } label: {
+                Image(systemName: "star.fill")
+                    .foregroundColor(.yellow)
+                Image(
+                    systemName: starSort == .starsAsc ? "chevron.up" :
+                        "chevron.down"
+                )
+            }
+            .sortButtonModifier()
+            Spacer()
+
+            Button {
+                if forkSort == .forksAsc {
+                    forkSort = .forksDes
+                } else {
+                    forkSort = .forksAsc
+                }
```

Comment:
this can also be written as a ternary operator, be consistent :)

Reply:
Awesome! 🥇